mantidproject/mantid
# Add temp correction convsimfit

**Description of work.**

This adds a temperature correction to the convolution fitting tab. 



**To test:**
  * I have sent a package to relevant scientists for them to functionally test. 
  * Here code review is needed.
  * It would also be good to test edge case inputs to check that such cases are handled gracefully.
  * Relevant data can be found in #26631



Fixes #26881  


*This does not require release notes* because this is only going into a feature branch. A full set of release notes covering the simultaneous fitting will be done later.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
